Fusion reactor

EldritchCrow opened the pull request:
Adding the fusion reactor

Entirely untested
Probably super OP in terms of output
May cause large server lag while reactor is operating, as the fusion effect runs pretty small checks every tick, and the temp of the gas in the chamber is 1e8

EldritchCrow committed:
Merge pull request #1 from Valencia-arch-project/master

Update from origin branch

EldritchCrow committed:
Everything for a fusion reactor

If something is broken, this probably caused it, since i have tested like none of it

davikhale commented:
Will pull this down to test locally before merging. Until then, standby.

EldritchCrow commented:
lemme provide instructions, its actually important
1. Make a circle of the magnets, surround with shielding if you want
2. Put a ton of heavy water into the center of the circle
3. Put a collector at the top of the circle, so its y - 1 is in the circle
4. Put a refractor in the center of the circle
5. Shoot muons at it the refractor
6. put wires going out of the refractor to see if it gives power

WARNING: Add sprites, everything will be invisible

EldritchCrow commented:
Now that I say all of this, lemme just test it

EldritchCrow commented:
ok i cant effectively test this without sprites, i have no idea whats going on because everything is either a wall or invisible

davikhale commented:
I will design placeholders tonight.
On Oct 13, 2015 6:33 PM, "MadSnailDisease" notifications@github.com wrote:

> ok i cant effectively test this without sprites, i have no idea whats
> going on because everything is either a wall or invisible
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Valencia-arch-project/valencia-arch1/pull/4#issuecomment-147883790
> .

EldritchCrow committed:
Adding a canister that holds deuterium/tritium mixture

Needs a sprite

EldritchCrow committed:
Adding config files bc git ext is borked

The .gitignore on the main repo will ignore these later, so they dont matter

EldritchCrow committed:
Adds circle finding code for use in fusion reactor

EldritchCrow committed:
Optimization of the fusion reactor code to reduce per tick CPU usage

I still need to test the ideas in java to identify some things for mapping, such as what the edges of the reactor should be
